src/dev-app/index.html

```diff
@@ -25,13 +25,30 @@
 <body>
   <dev-app>Loading...</dev-app>
 
+  <iframe id="google-maps-api-key"
```

Comment:
Could you add a comment explaining the use of this iframe? My interpretation is that this is a (neat) trick to load the API key without actually exposing the key on GitHub.

Reply:
Done.